src/runtime_src/core/common/api/xrt_kernel.cpp

```diff
@@ -119,8 +119,13 @@ xrtRunUpdateArgV(xrtRunHandle rhdl, int index, const void* value, size_t bytes);
 
 namespace {
 
-constexpr uint32_t MAILBOX_INPUT_CTRL  = (1 << 9);
-constexpr uint32_t MAILBOX_OUTPUT_CTRL = (1 << 10);
+constexpr uint32_t mailbox_input_write = 1;
+constexpr uint32_t mailbox_output_read = 1;
+constexpr uint32_t mailbox_input_ack = 2;
+constexpr uint32_t mailbox_output_ack = 2;
+constexpr size_t mailbox_auto_restart_cntr = 16; //0x10;
```

Comment:
I know, we are facing some issue if we use 0x10 instead of 16. Lets add a comment . We can fix it in next PR.